yarn.lock

```diff
@@ -28008,7 +27963,7 @@ webpack-virtual-modules@^0.4.1:
   resolved "https://registry.yarnpkg.com/webpack-virtual-modules/-/webpack-virtual-modules-0.4.3.tgz#cd597c6d51d5a5ecb473eea1983a58fa8a17ded9"
   integrity sha512-5NUqC2JquIL2pBAAo/VfBP6KuGkHIZQXW/lNKupLPfhViwh8wNsu0BObtl09yuKZszeEUfbXz8xhrHvSG16Nqw==
 
-webpack@4, webpack@^4.43.0:
+webpack@4:
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/192345947-367e898a-9083-4c0b-832f-d717d4f338b7.png)

![image](https://user-images.githubusercontent.com/14183168/192346004-25392956-cb0b-46a9-86e9-a061d5c38f29.png)

Wondering what still requires it.

Reply:
storybook :)

Reply:
storybook is very slow (known fact) as it re-builds its core on every run. Fun fact is that it also still uses webpack 4 to build itself. this will be partially mitigated in storybook 7.